# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/e17/evas/gl_common-fix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


GL_UNSIGNED_BYTE does produce incorrect (incorrectly byte-swapped) results
with both software rendering (MESA) and the radeon accelerator ...

  - Rene Rebe <rene@exactcode.de>

diff -ru evas-2005-05-05~/src/lib/engines/gl_common/evas_gl_private.h evas-2005-05-05/src/lib/engines/gl_common/evas_gl_private.h
--- evas-2005-05-05~/src/lib/engines/gl_common/evas_gl_private.h	2005-04-20 05:22:06.000000000 +0200
+++ evas-2005-05-05/src/lib/engines/gl_common/evas_gl_private.h	2005-05-05 23:32:29.000000000 +0200
@@ -84,7 +84,7 @@
 /* fast on vidia */
 /*#define NATIVE_PIX_UNIT   GL_UNSIGNED_INT_8_8_8_8_REV*/
 /* fast on ati compared to GL_UNSIGNED_INT_8_8_8_8_REV */
-#define NATIVE_PIX_UNIT   GL_UNSIGNED_BYTE
+#define NATIVE_PIX_UNIT   GL_UNSIGNED_INT_8_8_8_8_REV
 #endif
 
 #endif
